src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -82,6 +82,38 @@ object SampleWdl {
       """.stripMargin
   }
 
+  object CoercionNotDefined extends SampleWdl {
+    override def wdlSource(runtime: String = "") = {
+      """
+        |task summary {
+        |  command {
+        |    ~/plink --bfile ${bfile} --missing --hardy --out foo --allow-no-sex
+        |  }
+        |  output {
+        |    File hwe = "foo.hwe"
+        |    File log = "foo.log"
+        |    File imiss = "foo.imiss"
+        |    File lmiss = "foo.lmiss"
+        |  }
+        |  meta {
+        |    author: "Jackie Goldstein"
+        |    email: "jigold@broadinstitute.org"
+        |  }
+        |}
+        |
+        |
+        |workflow test1 {
+        |  String bfile
+        |  call summary {
+        |     input: bfile=bfile
+        |  }
+        |}
+      """.stripMargin
```

Comment:
So if I'm understanding this correctly, this WDL is supposed to be malformed?  If so, then I don't see how